tests/release/pipelines/release_to_github.go

```diff
@@ -0,0 +1,199 @@
+package pipelines
+
+import (
+	"encoding/json"
+	"fmt"
+//	"strings"
+
+	tektonutils "github.com/redhat-appstudio/release-service/tekton/utils"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	ecp "github.com/enterprise-contract/enterprise-contract-controller/api/v1alpha1"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	appservice "github.com/redhat-appstudio/application-api/api/v1alpha1"
+	appstudioApi "github.com/redhat-appstudio/application-api/api/v1alpha1"
+	"github.com/redhat-appstudio/e2e-tests/pkg/clients/has"
+	"github.com/redhat-appstudio/e2e-tests/pkg/constants"
+	"github.com/redhat-appstudio/e2e-tests/pkg/framework"
+	"github.com/redhat-appstudio/e2e-tests/pkg/utils"
+	"github.com/redhat-appstudio/e2e-tests/pkg/utils/contract"
+	"github.com/redhat-appstudio/e2e-tests/pkg/utils/tekton"
+	releasecommon "github.com/redhat-appstudio/e2e-tests/tests/release"
+	releaseApi "github.com/redhat-appstudio/release-service/api/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+var _ = framework.ReleasePipelinesSuiteDescribe("Push to external registry", Label("release-pipelines", "release-to-github"), func() {
+	defer GinkgoRecover()
+
+	var fw *framework.Framework
+	AfterEach(framework.ReportFailure(&fw))
+	var err error
+	var devNamespace, managedNamespace string
+
+	var component *appservice.Component
+	var releaseCR *releaseApi.Release
+
+	BeforeAll(func() {
+		// Initialize the tests controllers
+		fw, err = framework.NewFramework(utils.GetGeneratedNamespace("to-github"))
+		Expect(err).NotTo(HaveOccurred())
+		devNamespace = fw.UserNamespace
+		managedNamespace = utils.GetGeneratedNamespace("to-github-managed")
+		_, err = fw.AsKubeAdmin.CommonController.CreateTestNamespace(managedNamespace)
+		Expect(err).NotTo(HaveOccurred(), "Error when creating managedNamespace: %v", err)
+
+		sourceAuthJson := utils.GetEnv("QUAY_TOKEN", "")
```

Comment:
I'm not sure we need this, we don't push a snapshot in this pipeline

Reply:
Removed